src/include/daos_srv/pool.h

```diff
@@ -232,7 +232,8 @@ int ds_pool_svc_list_cont(uuid_t uuid, d_rank_list_t *ranks,
 
 int ds_pool_svc_check_evict(uuid_t pool_uuid, d_rank_list_t *ranks,
 			    uuid_t *handles, size_t n_handles,
-			    uint32_t destroy, uint32_t force);
+			    char *machine, uint32_t destroy,
+			    uint32_t force, uint32_t *count);
```

Comment:
Currently all the callers of the function assume anything non zero is an error. We can change it but its a larger change and the functions that make the call also assume a non-zero return code is an error so it would percolate pretty far up.

Reply:
No worries then. I'm fine with leaving as-is